api/v1beta1/types.go

```diff
@@ -126,6 +126,9 @@ type DOLoadBalancer struct {
 	// An object specifying health check settings for the Load Balancer. If omitted, default values will be provided.
 	// +optional
 	HealthCheck DOLoadBalancerHealthCheck `json:"healthCheck,omitempty"`
+	// The DO load balancer uuid. If omitted, a new load balancer will be created.
+	// +optional
+	ResourceId string `json:"resourceId,omitempty"`
```

Comment:
I _think_ we may need to add this field to the struct in `v1alpha4/types.go` too. It's likely what's causing the conversion warning you're seeing when running `make generate`

Reply:
Possibly `v1alpha3/types.go` too

Reply:
> What's our policy on updating APIs as old as v1alpha3? 

I'm not 100% sure. Maybe you're right that worrying about v1alpha3 is unnecessary. @cpanato or @prksu might know for sure? Either way it'd be good to have an explicit rule regarding that.

Reply:
@cpanato need your 👀 ^^

Reply:
for olders api, i think we don't need to do that, otherwise might break users, but in the conversion we should add that

@fabriziopandini it is the correct approach when adding a new field in the newer API and not in the older ones? and just write some conversions? or what is the best practice for that? thanks in advance